fw/ja5_conf.c

```diff
@@ -0,0 +1,190 @@
+/**
+ *		Tempesta FW
+ *
+ * Transport Layer Security (TLS) interfaces to Tempesta TLS.
+ *
+ * Copyright (C) 2024 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+#include "ja5_conf.h"
+#include "hash.h"
+
+#define TLS_JA5_DEFAULT_MAX_ENTRIES 100
+#define TLS_JA5_HASHTABLE_BITS 10
+
+typedef struct {
+	u32 max_entries_cnt;
```

Comment:
Where we are planning to use `max_entries_cnt`? As I think we should check that count of hash entries is not greater then `max_entries_cnt` isn't it?